aspnetcore/tutorials/first-mvc-app-xplat/new-field.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Adding a New Field | Microsoft Docs
+author: rick-anderson
+description: 
+keywords: ASP.NET Core,
+ms.author: riande
+manager: wpickett
+ms.date: 04/14/2017
+ms.topic: article
+ms.assetid: 1638bacf-fe7b-4b41-84b0-06a1574b7734
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: tutorials/first-mvc-app-xplat/new-field
+---
+# Adding a New Field
+
+By [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+When you use EF Code First to automatically create a database, Code First adds a table to the database to help track whether the schema of the database is in sync with the model classes it was generated from. If they aren't in sync, EF throws an exception. This makes it easier to find inconsistent database/code issues.
+
+This tutorial will add a new field to the `Movies` table. One option is to use [EF Code First Migrations](http://docs.efproject.net/en/latest/platforms/aspnetcore/new-db.html) to add a new field to the model and migrate that change to the database - but SQLlite does not support many migration schema operations, so only very simply migrations are possible. See [SQLite Limitations](https://docs.microsoft.com/ef/core/providers/sqlite/limitations) for more information.
```

Comment:
Consider presenting here what you're going to do, i.e., drop and re-create the database, then in a separate paragraph, discuss migrations, ending with a link to that limitations/workarounds doc.  As it stands, it's a little confusing hearing about migrations, then we're not going to use it, then we're presented with options for migrations workarounds.

An alternative is to put the migrations discussion below after #3 in the "There are a few approaches to resolving the error."  Would be a good place to both introduce migrations and explain why you're not using it.